src/events/autoMod.js

```diff
@@ -0,0 +1,24 @@
+const fs = require("fs");
+
+class CacheManager {
+     constructor(val) {
+          this.cache = String(val).split(/[\n\r]/);
+          this.cache = this.cache.filter((url) => url.length > 0);
+          this.cache = new Map(this.cache.map((url) => [url, true]));
```

Comment:
a lot better!
maybe don't write all of them in this.cache repeatedly, that looks confusing IMO, just the last value? 
the ones before can either be `const`s, or even a oneliner if you feel like it

Reply:
I don't see any other way, how would we cache only last line? I mean it would only check for one url :D

Reply:
I meant something like
`const splitStrings = ..
const filteredArray = ..
this.cache= mapFrom(filteredArray)`